abstract/group.go

```diff
@@ -51,6 +51,9 @@ type Secret interface {
 
 	// Set to a fresh random or pseudo-random secret
 	Pick(rand cipher.Stream) Secret
+
+	// SetBytes will take bytes and create a secret out of it
+	SetBytes([]byte) Secret
```

Comment:
I totally understand all the **semantics** and **practical** differences between nist.Int and big.Int but even though, from an **API perspective**, I should have said, it still *feels wrong*.  `(Set)Bytes` feels like it's a getter/setter interface while it's not because it introduces change (modulo) in the middle. Moreover, the `Bytes()` functionality can be questioned regarding its utility because it can be emulated using `MarshalBinary` and then cut off the padding.

Well, anyway, it's an endless loop and I think we should move on, so I accept this solution without any regret ;)

Regarding the naming, yes it would be a good idea to rename it `ModInt` or `IntMod` stg along those lines...